edx_theme/breadcrumbs.html

```diff
@@ -3,9 +3,9 @@
   <li><a href="">{{ title }}</a></li>
   <li class="wy-breadcrumbs-aside">
     {%- if help_url -%}
-      <a href="{{ help_url }}" target="_blank">Need Help?</a> |
+      <a href="{{ help_url }}" target="_blank">{{ help_link_text }}</a> or
     {%- endif -%}
-    <a href="{{ feedback_form_url }}" target="_blank">Feedback</a>
+    <a href="{{ feedback_form_url }}" target="_blank">{{ feedback_form_link_text }}</a>
```

Comment:
Should we have a default `html_context` that gives values for these when not specified in the project using the theme?